apps/frontend/src/library/ResearchOutput.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { ResearchOutputPage, Paragraph } from '@asap-hub/react-components';
+import { useResearchOutputById } from '../api';
+
+const ResearchOutput: React.FC = () => {
+  const { id } = useParams();
+  const { loading, data: researchOutputData, error } = useResearchOutputById(
+    id,
+  );
+
+  if (loading) {
+    return <Paragraph>Loading...</Paragraph>;
+  }
+
+  if (researchOutputData) {
+    const researchOutput = {
+      ...researchOutputData,
+      team: researchOutputData.team
+        ? {
+            ...researchOutputData.team,
+            href: `/network/teams/${researchOutputData.team.id}`,
+          }
+        : undefined,
+      profileHref: '#',
```

Comment:
Don't know where to link this

Reply:
Yeah that's unfortunate, @fampinheiro shouldn't the research output (response) have some sort of author?

Reply:
Oh this effectively appears to be the `ASAP Admin` thing?

Reply:
Do we know where that should go @fampinheiro ?

Reply:
The author is a left-over from ROMP, on this case it should link to a section on the website about ASAP.
Can be a simple page like `terms-and-conditions`, but for now will point nowhere :(

One more pending link to add to the list.

Reply:
We do not. Will put into a backlog ticket for when we do